common/supplemental/supplementalData.xml

```diff
@@ -1396,7 +1396,7 @@ XXX Code for transations where no currency is involved
 		<language type="bku" scripts="Latn"/>
 		<language type="bku" scripts="Buhd" alt="secondary"/>
 		<language type="bla" scripts="Latn"/>
-		<language type="blo" scripts="Latn" territories="BJ TG"/>
```

Comment:
this may have been added, manually, since Anii (blo) is in CLDR. Was there any warning when you ran the generator?

Reply:
Yes, as described in the ticket:
```
# Problems in language_script.tsv

* DETAIL *	changing <languageData>:	blo	Anii	[BJ:Benin]  REMOVED!
* DETAIL *	changing <languageData>:	blo	Anii	[TG:Togo]  REMOVED!
* DETAIL *	changing <languageData>:	blo	Anii	{primary=[primary; scripts=Latn; territories=]}
```

Reply:
That shouldn't cause a problem. blo is still present.